### ISSUE TITLE
{Freeway-Bug} [Adaptive Cards> CalendarReminder.json]: No mechanism is provided to the user to switch between the module window to preview window.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7827]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25182.1010)
Browser: Edge Dev (106.0.1356.0)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2206.2001
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ windows+ enter key.
3. Navigate to the 'ActivityUpdate.Json' control through tab key and activate it.
4. Now, navigate through tab key to the 'Publish Adaptive Card Schema' card.
5. Observe the issue.

**Actual Result:**
Any mechanism or key bind command short cut is not defined to switch between the module window to preview window. As user has to press many times tab key to navigate on module window to preview window.

**Expected Result:**
Any mechanism or key bind command short cut should be defined so that user can switch between the module window to preview window.

**Observation:**
Same issue repro throughout the application.

**User Impact:**
Person with cognitive disability will get impacted if every time they have to press tab key multiple times to navigate between modules which crete difficulty for them in navigating the control or in making track of the content properly.

**MAS Reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.01.04

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/104483005/189747651-1237340b-4ab8-4868-8c10-bf13879e76f6.mp4

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @vaishali1397,

I'm closing this issue because we do not release the UWP visualizer as an app for customers to use. We use this app for internal testing. Only accessibility issues found in the preview window will reach our customers because these are caused by the Adaptive Cards UWP rendering package.

Please let me know if you have any questions - thanks!

**Comment by vaishali1397** [comment]
Closing this bug as per below comment.
![image](https://user-images.githubusercontent.com/104483005/212275602-d4c98519-6542-46c7-b941-d9044c753e03.png)
